packages/sanity/playwright-ct/tests/formBuilder/inputs/PortableText/FocusTracking.spec.tsx

```diff
@@ -136,6 +137,24 @@ test.describe('Portable Text Input', () => {
       await expect(blockObjectInput).not.toBeVisible()
     })
   })
+  test(`reports focus on spans with with .text prop, and everything else without`, async ({
+    mount,
+    page,
+  }) => {
+    const paths: Path[] = []
+    const pushPath = (path: Path) => paths.push(path)
+    await mount(<FocusTrackingStory document={document} onPathFocus={pushPath} />)
+    const {getFocusedPortableTextEditor} = testHelpers({page})
+    const $pte = await getFocusedPortableTextEditor('field-body')
```

Comment:
Not a huge deal, but is there any particular meaning behind the `$`-prefix here? I guess it's because these refer to DOM elements, but my impression is that it's not a commonly used convention these days, since you can easily tell from the type anyway(?)

Reply:
It's to signal that it's a (synchronous) [Locator](https://playwright.dev/docs/locators) instance.
The test suite started with this convention, and I think the Playwright documentation used to have this convention in their examples, but I can't find them anymore. I'm all in for not using them, but then we should follow the same convention everywhere within our test suite.